0-ad-history-mod-package/simulation/templates/gaia/fauna_shark.xml

```diff
@@ -13,6 +13,10 @@
     <SpecificName>Carcharodon carcharias</SpecificName>
     <Classes datatype="tokens">SeaCreature</Classes>
     <Icon>gaia/fauna_shark.png</Icon>
+    <History>
+        The Great White Shark (sometimes also known as the White Pointer or just the White Shark) is probably the most fearsome predator in the oceans. Its length ranges from 20 to 28 feet and it can weigh up to 2 and ¼ tons. It’s skin is gray, and it has a white underbelly It’s powerful body allows it to chase down its prey and to devour them with its many rows of jagged teeth. Teeth that come in row by row thorough out its entire life.
```

Comment:
```suggestion
        The Great White Shark (sometimes also known as the White Pointer or just the White Shark) is probably the most fearsome predator in the oceans. Its length ranges from 20 to 28 feet, and it can weigh up to 2 ¼ tons. Its skin is gray, and it has a white underbelly. Its powerful body allows it to chase down its prey and devour them with its many rows of jagged teeth. Teeth that come in row by row throughout its entire life.
```